ui/src/components/song/SongListContainer.tsx

```diff
@@ -26,49 +27,102 @@ const SongListContainer: FC = (): ReactElement => {
   const [open, setOpen] = useState(false);
   const [showDetails, setShowDetails] = useState(true);
 
+  const [page, setPage] = useState(1);
+  const [totalPages, setTotalPages] = useState(1);
+
   const isDesktop = useMediaQuery('(min-width: 768px)');
   const navigate = useNavigate();
   const location = useLocation();
-
   const handleClose = () => setOpen(false);
-
   const allSongs = useSongs() as SongSchema[];
 
+  const [loading, setLoading] = useState(false);
+
+  const handleScroll = () => {
```

Comment:
replicate this search behavior in the setlist view. there is also a song search there when trying to add/edit setlist.

See `src/components/setlist/SetlistEditorContainer.tsx`

Reply:
i will do it since i'm working on setlist pages :)